fpdf/fpdf.py

```diff
@@ -3168,6 +3225,19 @@ def frag():
                     gstate["text_color"] = self.MARKDOWN_LINK_COLOR
                 yield Fragment(list(link_text), gstate, self.k, url=link_url)
                 continue
+            if ttf_font and txt[0] != "\n" and not ord(txt[0]) in font_glyphs:
+                fallback_font = self._get_fallback_font(txt[0])
+                if fallback_font:
+                    if txt_frag:
+                        yield frag()
+                    gstate = self._get_current_graphics_state()
+                    gstate["font_family"] = fallback_font
+                    yield Fragment(txt[0], gstate, self.k)
+                    txt = txt[1:]
+                    continue
+                raise EnvironmentError(
```

Comment:
Seems like this error should only be raised if fallback fonts are defined?
Otherwise a warning is already generated, _cf._ #661

Also, maybe a `FPDFException` would be a better choice of class